### PR TITLE
boards: stm32f769i_disco: Add SDMMC support

### PR DESCRIPTION
**Pull request by ExtremeGTX**
- Define sdmmc2 for stm32f7 series.
- Define board specific config for sdmmc2 module.

Tested using `zephyr/samples/subsys/fs/fat_fs/` and MicroSD Card formatted as FAT32.
 
Signed-off-by: Mohamed ElShahawi <ExtremeGTX@hotmail.com>

### CODE REVIEW

File: dts/arm/st/f7/stm32f7.dtsi [hunk 1]
```diff
@@ -679,6 +679,14 @@
 			status = "disabled";
 			label = "SDMMC_1";
 		};
+
+		sdmmc2: sdmmc@40011c00 {
```
Comment (erwango) [inline dts/arm/st/f7/stm32f7.dtsi:683]: SDMMC2 is not available for stm32f74x/stm32f75x, so please move node definition in stm32f72x/stm32f76x

**Review by erwango — CHANGES_REQUESTED** [review]
Thanks for this contribution!

Aside from the provided comment, can yo split the PR in 2 commits:
- dts: Add ...
- board: Add ...

Txs

File: dts/arm/st/f7/stm32f7.dtsi (hunk 1 above)
Comment (ExtremeGTX) [reply dts/arm/st/f7/stm32f7.dtsi:683]: Is there a table to quickly check the features per SoC ?

Comment (erwango) [reply dts/arm/st/f7/stm32f7.dtsi]: > Is there a table to quickly check the features per SoC ?

I'm usually comparing CMSIS files (https://github.com/zephyrproject-rtos/hal_stm32/tree/master/stm32cube/stm32f7xx/soc)

**Review by erwango — APPROVED** [review]

**Comment by ExtremeGTX** [comment]
@ABOSTM @FRASTM Can you please review and approve ?

**Review by ABOSTM — APPROVED** [review]
LGTM

**Pull request merged**